Jellyfin.Server/Migrations/PreStartupRoutines/CreateNetworkConfiguration.cs

```diff
@@ -114,9 +114,7 @@ public string BaseUrl
 
         public bool IgnoreVirtualInterfaces { get; set; } = true;
 
-        public string VirtualInterfaceNames { get; set; } = "vEthernet*";
-
-        public bool TrustAllIP6Interfaces { get; set; }
+        public string[] VirtualInterfaceNames { get; set; } = new string[] { "veth" };
```

Comment:
What about `vEthernet*`?

Reply:
It's also matched with this. The function facilitating it matches every interface beginnin with whatever you put here (this was also the case before)

Reply:
Fair